core/templates/pages/contributor-dashboard-page/modal-templates/translation-suggestion-review-modal.component.spec.ts

```diff
@@ -1558,4 +1558,15 @@ describe('Translation Suggestion Review Modal Component', function () {
       expect(component.doesTranslationContainTags()).toBeFalse();
     });
   });
+
+  it('should extract html related to images', () => {
+    const content =
+      '<oppia-noninteractive-image src="image1.jpg">something</oppia-noninteractive-image><another-tag value="a"></another-tag><oppia-noninteractive-image src="image2.jpg"></oppia-noninteractive-image>';
+    const expectedHtmlString =
+      '<oppia-noninteractive-image src="image1.jpg">something</oppia-noninteractive-image><oppia-noninteractive-image src="image2.jpg"></oppia-noninteractive-image>';
```

Comment:
Can you split this over multiple lines to keep to shorter than 80 chars?

Reply:
Done!